index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Andrew">
+    <link rel="stylesheet" href="style.css">
+    <title>basic structure practice</title>
+</head>
+<body>
+
+    <!-- my main heading -->
+    
+<h1 class="">this is the most important heading</h1>
+
+<!-- paragraph tag  -->
+    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Facilis sed dolor dolorem. </p>
+
+    <!-- my sub-main heading  -->
+    
+<h2>thi heading is a little less important</h2>
+
+    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos illo labore quas nisi, earum dignissimos ipsa mollitia at? Architecto ut </p>
+
+
+    <!-- my last sub-heading -->
+<h3>this heading is even less important than the one above</h3>
+
+    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos illo labore quas nisi, earum dignissimos ipsa mollitia at? Architecto ut</p>
+    
+
+    /**********************************
+    *__        __   _     _   _       *
+    *\ \      / /__| |__ | | | |_ __  *
+    * \ \ /\ / / _ \ '_ \| | | | '_ \ *
+    *  \ V  V /  __/ |_) | |_| | |_) |*
+    *   \_/\_/ \___|_.__/ \___/| .__/ *
+    *                          |_|    *
+    ***********************************/
+
```

Comment:
You need to wrap the content above in a `<pre>` tag to preserve the pre-formatted text